web/index.html

```diff
@@ -25,7 +25,8 @@
           <form class="form-inline my-2 my-lg-0">
             <input class="form-control mr-sm-2" id="email" type="email" placeholder="email address" aria-label="email address">
             <button onClick="accessAccount()" class="btn btn-secondary my-2 my-sm-0"><i class="fas fa-arrow-left"></i><i class="fas fa-envelope"></i> Access account</button> &nbsp;
-            <button onClick="generateAccount()" id="btn-gen-random" class="btn btn-secondary my-2 my-sm-0"><i class="fas fa-random"></i> Generate random</button>
+            <button onClick="generateAccount()" id="btn-gen-random" class="btn btn-secondary my-2 my-sm-0"><i class="fas fa-random"></i> Generate random</button> &nbsp;
+            <button onClick="listAddresses(event)" id="btn-list-addresses" class="btn btn-secondary my-2 my-sm-0"><i class="fas fa-list"></i> List accounts</button>
```

Comment:
please make it hidden by default and only show the button when it's activated. this way it won't flash when the page loads.

You can do it via CSS since you already have the ID `btn-list-addresses`